samples/bluetooth/broadcaster_multiple/src/broadcaster_multiple.c

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/printk.h>
+#include <zephyr/bluetooth/bluetooth.h>
+
+/* Maximum supported AD data length, use a value supported by the Controller,
+ * Bluetooth Core Specification define minimum of 31 bytes will be supported by
+ * all Controllers, can be a maximum of 1650 bytes when supported.
+ */
+#if defined(CONFIG_BT_CTLR_ADV_DATA_LEN_MAX)
+#define BT_AD_DATA_LEN_MAX CONFIG_BT_CTLR_ADV_DATA_LEN_MAX
+#else
+#define BT_AD_DATA_LEN_MAX 31U
+#endif
+
+/* Size of AD data format length field in octets */
+#define BT_AD_DATA_FORMAT_LEN_SIZE 1U
+
+/* Size of AD data format type field in octets */
+#define BT_AD_DATA_FORMAT_TYPE_SIZE 1U
+
+/* Maximum value of AD data format length field (8-bit) */
+#define BT_AD_DATA_FORMAT_LEN_MAX 255U
+
+/* Device name length, size minus one null character */
+#define BT_DEVICE_NAME_LEN (sizeof(CONFIG_BT_DEVICE_NAME) - 1U)
+
+/* Device name length in AD data format, 2 bytes for length and type overhead */
+#define BT_DEVICE_NAME_AD_DATA_LEN (BT_AD_DATA_FORMAT_LEN_SIZE + \
+				    BT_AD_DATA_FORMAT_TYPE_SIZE + \
+				    BT_DEVICE_NAME_LEN)
+
+/* Maximum manufacturer data length, considering ad data format overhead and
+ * the included device name in ad data format.
+ */
+#define BT_MFG_DATA_LEN_MAX       (MIN((BT_AD_DATA_FORMAT_LEN_MAX - \
+					BT_AD_DATA_FORMAT_TYPE_SIZE), \
+				       (BT_AD_DATA_LEN_MAX - \
+					BT_AD_DATA_FORMAT_LEN_SIZE - \
+					BT_AD_DATA_FORMAT_TYPE_SIZE)))
+#define BT_MFG_DATA_LEN           (MIN(BT_MFG_DATA_LEN_MAX, \
+				       (BT_AD_DATA_LEN_MAX - \
+					BT_AD_DATA_FORMAT_LEN_SIZE - \
+					BT_AD_DATA_FORMAT_TYPE_SIZE - \
+					BT_DEVICE_NAME_AD_DATA_LEN)))
+
+static uint8_t mfg_data[BT_MFG_DATA_LEN] = { 0xFF, 0xFF, };
+
+static const struct bt_data ad[] = {
+	BT_DATA(BT_DATA_MANUFACTURER_DATA, mfg_data, sizeof(mfg_data)),
+};
```

Comment:
Did you consider adding more entries to this list to demonstrate how to send more than 255 bytes of data?

Reply:
I did manually test it by added more `BT_DATA_MANUFACTURER_DATA` entries, but not updated here in the sample, as the calculation of the subsequent size of the array and conditional compilation of additional entries here is complicated when it is to be derived from the `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX` value.